src/plugins/data/common/search/es_search/types.ts

```diff
@@ -46,4 +46,11 @@ export interface IEsSearchRequest extends IKibanaSearchRequest<ISearchRequestPar
   indexType?: string;
 }
 
+export interface IEsRawSearchResponse<Source = any> extends SearchResponse<Source> {
+  id?: string;
+  is_partial?: boolean;
+  is_running?: boolean;
+  response?: SearchResponse<Source>;
```

Comment:
Since `response` is only defined when using async search, I think we should leave this out and make a new type for the async search response.